config/01-ce-router-defaults.conf.in

```diff
@@ -49,23 +36,22 @@ SYSTEM_PERIODIC_HOLD_REASON = \
      ) \
    )
 
-# Remove jobs that have been on hold for longer than 24 hrs
-SYSTEM_PERIODIC_REMOVE = (JobStatus == 5 && CurrentTime - EnteredCurrentStatus > 3600*24)
+# Remove jobs that have been on hold for longer than 24 hrs or held jobs that have already started once
+SYSTEM_PERIODIC_REMOVE = (JobStatus == 5 && CurrentTime - EnteredCurrentStatus > 3600*24) || JobStatus == 1 && NumJobStarts == 1) || (NumJobStarts > 1)
```

Comment:
@JaimeFrey I was hoping for your opinion on this periodic remove expression for disabling job retries on the CE. Two things that I can think of off the top of my head:

1. We should only remove CE jobs (i.e. not routed grid universe jobs) based on number of job retries
1. We may want to avoid removing held jobs with `NumJobStarts > 1` so if there are helpful hold messages we don't remove them from under the admin immediately

Reply:
Moving discussion to #242